build/scripts/build.ps1

```diff
@@ -642,6 +642,7 @@ function Redirect-Temp() {
     Copy-Item (Join-Path $repoDir "src\Workspaces\CoreTestUtilities\Resources\.editorconfig") $temp
```

Comment:
nit: I feel like these should be part of a build target, so users aren't required to use the build script to get "good" behavior.

Reply:
At one point we had us writing these files in unit tests...

Reply:
I don't think writing them in the unit tests is desirable, but possibly as a `Copy` task in one of the root `Test.Utilities` projects....